### PR TITLE
Minor changes throughout Best Practices for Reusable Bundles

### PR DESCRIPTION
**Pull request by darrylhein**
Cleanup and fixes for parts that were difficult to understand

### CODE REVIEW

File: bundles/best_practices.rst [hunk 1]
```diff
@@ -75,7 +76,7 @@ configuration options (see below for some usage examples).
 Directory Structure
 -------------------
 
-The basic directory structure of an AcmeBlogBundle must read as follows:
+The basic directory structure of an ``AcmeBlogBundle`` must read as follows:
```
Comment (xabbuh) [inline bundles/best_practices.rst]: I am against this change. We usually do not enclose bundle names with backticks.

File: bundles/best_practices.rst [hunk 2]
```diff
@@ -105,15 +106,15 @@ that automated tools can rely on:
...
+The depth of subdirectories should be kept to a minimum for the most used
+classes and files. Two levels is the maximum.
 
 The bundle directory is read-only. If you need to write temporary files, store
 them under the ``cache/`` or ``log/`` directory of the host application. Tools
 can generate files in the bundle directory structure, but only if the generated
 files are going to be part of the repository.
 
-The following classes and files have specific emplacements (some are mandatory
+The following classes and files have specific emplacements (some are mandatory
```
Comment (xabbuh) [inline bundles/best_practices.rst]: This change should be reverted.

File: bundles/best_practices.rst [hunk 3]
```diff
@@ -404,10 +404,10 @@ The ``composer.json`` file should include at least the following metadata:
 
 ``license``
     ``MIT`` is the preferred license for Symfony bundles, but you can use any
-    other license.
+    license.
 
 ``autoload``
-    This information is used by Symfony to load the classes of the bundle. The
+    This information is used by Symfony to load the classes in the bundle. The
```
Comment (xabbuh) [inline bundles/best_practices.rst]: This change looks wrong to me. Autoloading is about how classes provided by the bundle are loaded. So we are actually talking about "classes **of** the bundle".

File: bundles/best_practices.rst (hunk 1 above)
Comment (wouterj) [inline bundles/best_practices.rst]: I agree with @xabbuh here. It's a name and not a code-segment. Plus, any changes in text rendering is a minor break of the read flow, so having less in-line code blocks will improve docs readability.

File: bundles/best_practices.rst [hunk 4]
```diff
@@ -28,10 +28,11 @@ the guides.
 Bundle Name
 -----------
 
-A bundle is also a PHP namespace. The namespace must follow the `PSR-0`_ or
-`PSR-4`_ interoperability standards for PHP namespaces and class names: it starts
-with a vendor segment, followed by zero or more category segments, and it ends
-with the namespace short name, which must end with a ``Bundle`` suffix.
+A bundle name is also a PHP namespace. The namespace must follow the `PSR-0`_
```
Comment (wouterj) [inline bundles/best_practices.rst]: I'm against this change, the bundle is a PHP namespace. Not the bundle name perse.

File: bundles/best_practices.rst (hunk 1 above)
Comment (darrylhein) [inline bundles/best_practices.rst]: Should we then remove the backticks from the previous section? There are 3 locations where the bundle name is in backticks. ~25% of the references to AcmeBlogBundle on this page are not in backticks while the rest are. (Again, just going for consistency, but don't really care.)

File: bundles/best_practices.rst (hunk 4 above)
Comment (darrylhein) [inline bundles/best_practices.rst]: I think my reasoning was twofold: (1) the section is called "Bundle Name"; (2) the bundle contains/has a namespace whereas the namespace isn't the bundle. I was confused when reading it (being new to bundles and semi-new to namespaces) as to how the bundle became a namespace. Would it make more sense to do the following: _The bundle name is used as the PHP namespace._

File: bundles/best_practices.rst [hunk 5]
```diff
@@ -185,7 +186,7 @@ All classes and functions must come with full PHPDoc.
 
 Extensive documentation should also be provided in the
 :doc:`reStructuredText </contributing/documentation/format>` format, under
-the ``Resources/doc/`` directory; the ``Resources/doc/index.rst`` file is
+the ``Resources/doc/`` directory. The ``Resources/doc/index.rst`` file is
```
Comment (OskarStark) [inline bundles/best_practices.rst]: does it really needs to be RST?

File: bundles/best_practices.rst [hunk 6]
```diff
@@ -404,7 +404,7 @@ The ``composer.json`` file should include at least the following metadata:
 
 ``license``
     ``MIT`` is the preferred license for Symfony bundles, but you can use any
-    other license.
+    license.
```
Comment (OskarStark) [inline bundles/best_practices.rst]: :+1:

File: bundles/best_practices.rst (hunk 5 above)
Comment (darrylhein) [inline bundles/best_practices.rst]: @OskarStark @stof I've reworded this section. Does this help?

**Review by javiereguiluz — APPROVED** [review]
:+1:

I made all the changes suggested by reviewers to make this mergeable.

**Review by HeahDude — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @darrylhein.

**Pull request closed without merging**